admin_manual/appliance/firewall.rst

```diff
@@ -0,0 +1,19 @@
+Firewall protected environment
+------------------------------
+
+If you are considering setting up the appliance in an environment with a firewall, 
+please create rules that permit access to the following hosts. 
+If your DNS is not working, you can use the IP addresses instead.
+If you are using google as your DNS server(IP=8.8.8.8), you have to permit access to it too.
+
+
+Firewall Rules::
+
+	software-univention.de
+	owncloud.org
+	owncloud.com
+	docker.software-univention.de
+	176.9.114.147
+	marketplace.owncloud.com
+	5.9.68.237
+  	8.8.8.8
```

Comment:
Well... I wrote only that what I entered to get it working. I had trouble with the DNS so I entered the IP address instead. For people who have similar issues this would be a good guide. It's not really about hiding :)